src/components/submission/list.vue

```diff
@@ -276,6 +276,13 @@ export default {
       })
         .finally(() => { this.refreshing = false; })
         .catch(noop);
+
+      // refresh keys when refreshing submissions (keys initially loaded in form/submissions)
+      if (refresh) {
```

Comment:
> If there isn't encryption, then we can know there isn't a managed key.

Actually, if the project is encrypted in another tab, then it could be the case that `this.formVersion.keyId == null` even while there are encryption `keys` to fetch. That case is described in the issue. However, I still think the optimization here is a good idea. The issue suggested that we don't need to address that case.